### DISCUSSION TITLE
Synchronize keyboard input between panes?

### DISCUSSION THREAD

**Discussion by jmcastelo** [wezterm/wezterm#1602] [Q&A]
Is there a way to synchronize the panes so that whatever one types on the keyboard goes to all displayed panes of the current tab?

This is possible with tmux and find it useful when issuing commands simultaneously to different computers connected via ssh in the same tab and different panes.

**Comment by wez** [comment]
Not today, no!

**Reply by jmcastelo** [reply]
Is there any chance this will be implemented in the future?
Thanks for the reply, and for fantastic wezterm!

**Reply by wez** [reply]
I have no plans to implement it myself, but I'm open to someone adding that feature in a PR
